doc/tutorials/namespaces.md

```diff
@@ -48,7 +48,7 @@ Now we abandon the file system analogy and introduce the concept of _namespaces_
 Every key in Elektra belongs to one of these namespaces:
 
 - **spec** for specification of other keys
-- **proc** for in-memory keys (e.g. command-line)
+- **proc** for in-memory keys (e.g. command line)
```

Comment:
Actually the original way seems to be correct, see: https://en.wikipedia.org/wiki/Command-line_interface

You say "command-line" if there is a following noun that gets referred to. Otherwise, it is simply "command line".

Both in Elektra and the Wikipedia article there is incorrect use of: "command line interface", maybe you can fix that, instead?

Reply:
You are completely right the CLI( command line interface) has no hyphen in it, I will revert the changes and change the wrong command line interface.

Reply:
Ohh, you didn't see the update in my message above, sorry for the confusion.

Your change here was okay, the "command line interface" at other spots is wrong...